File: CS/WindowsFormsApplication1/MyDockManager/MyPaintStyle.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars.Docking.Paint;
using DevExpress.XtraBars.Styles;
using DevExpress.XtraBars.Docking;

namespace WindowsFormsApplication1
{
    public class MyPaintStyle : SkinBarManagerPaintStyle
    {

        public MyPaintStyle(BarManagerPaintStyleCollection collection)
            : base(collection)
        {
        }
        public override string Name { get { return "MyScheme"; } }
        protected override void CreatePainters()
        {
            base.CreatePainters();
            fElementsPainter = new MyDockElementsSkinPainter(this);
        }


    }
}

File: VB/WindowsFormsApplication1/MyDockManager/MyPaintStyle.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars.Docking.Paint
Imports DevExpress.XtraBars.Styles
Imports DevExpress.XtraBars.Docking

Namespace WindowsFormsApplication1
	Public Class MyPaintStyle
		Inherits SkinBarManagerPaintStyle

		Public Sub New(ByVal collection As BarManagerPaintStyleCollection)
			MyBase.New(collection)
		End Sub
		Public Overrides ReadOnly Property Name() As String
			Get
				Return "MyScheme"
			End Get
		End Property
		Protected Overrides Sub CreatePainters()
			MyBase.CreatePainters()
			fElementsPainter = New MyDockElementsSkinPainter(Me)
		End Sub


	End Class
End Namespace
